src/PhpSpec/Wrapper/Subject/Expectation/DuringCall.php

```diff
@@ -95,21 +95,30 @@ public function __call(string $method, array $arguments = array())
             return $this->during(lcfirst($matches[1]), $arguments);
         }
 
-        throw new MatcherException('Incorrect usage of matcher Throw, '.
-            'either prefix the method with "during" and capitalize the '.
-            'first character of the method or use ->during(\'callable\', '.
-            'array(arguments)).'.PHP_EOL.'E.g.'.PHP_EOL.'->during'.
-            ucfirst($method).'(arguments)'.PHP_EOL.'or'.PHP_EOL.
-            '->during(\''.$method.'\', array(arguments))');
+        throw new MatcherException(
```

Comment:
I don't consider this use of `sprintf` more readable

Reply:
Ok, will submit the change tonight, gotta work now :(

Reply:
I've done it but have left your commit

Reply:
It's fine :-)